Python/auto_update_quarto.py

```diff
@@ -0,0 +1,35 @@
+"""
+This script will download and install the latest version of Quarto.
```

Comment:
```suggestion
This script will download and install the latest version of Quarto.
Useful because Quarto is not provided by Ubuntu's package manager.
```
I was wondering why the script is useful in the first place, but I guess this is the reason?

Reply:
Yeap, they just provide a `.deb` file. You cannot fetch it from a repository.